### ISSUE TITLE
Testimonials section

### ISSUE THREAD

**Issue by mkevenaar** [chocolatey/chocolatey.org#636]
Hi team,

It would be great to have a testimonials section on the chocolatey.org website. All users should be able to submit a testimonial including FOSS users

**Comment by ferventcoder** [comment]
Agreed. It's something we've wanted to do for awhile, because third party validation is awesome. 👍
